api-docs-v1.md

```diff
@@ -82,6 +82,16 @@
   **Content:** `{ error : error : "You are unauthorized to make this request." }`
```

Comment:
@vinay8494 we need two "error"s here and all the responses?

Reply:
no we dont' typo